### DISCUSSION TITLE
I failed to register for the student pack

### DISCUSSION THREAD

**Discussion by yibajianghudao** [community/community#53750] [GitHub Education]
I was disappointed to receive the failed message, most likely because I didn't use the EDU email and put a space after my school name, because my school did not register new students with the EDU mailbox (which is terrible), but if you choose my school you must use the EDU email address to apply, so I used this stupid method.And I had to send the application through the VPN old, but he would make my address be located in the wrong place, I have a lot of ID photos to prove my identity, but only allowed me to upload a photo, I tried to group them together, but he looked blurry and I felt terrible.
I have resubmitted the application, hoping to get the news of success.

**Comment by samus-aran** [comment]
Hi @yibajianghudao 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

Let us know how you get on 👍

**Reply by yibajianghudao** [reply]
![image](https://user-images.githubusercontent.com/111487502/234737726-377b257b-39a7-4c98-8dab-8deb0ed00f4b.png)
![image](https://user-images.githubusercontent.com/111487502/234737763-fc732f13-18f8-4d5b-ad90-b628a6b7bc96.png)
  After the first application was rejected, I solved the feedback problem received, but the second application was directly rejected without specifying the name of the school. As I just mentioned, my school must use the edu mailbox to register, but currently it is not No edu mailboxes were issued to our cohort (although we have been enrolled for almost a year), so I had to put a space after my school name (I thought the reviewers could tell) and I ended up submitting a lot of materials , including my student ID card, campus card, class schedule, student certification form, and contains English translation and original copy, but I still failed, which makes me very distressed, I need the student status of github.
  Hope someone can help I, thank you.
